packages/paste-core/components/chat-log/src/ChatMessage.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+
+import type {MessageVariants, ChatMessageProps} from './types';
+import {messageVariantStyles} from './styles';
+
+export const MessageVariantContext = React.createContext<MessageVariants>('inbound');
```

Comment:
question: I like how you're using react context here, it totally makes sense, I was just wondering if doing it this way would be any different from passing `variant` down to children using a `React.cloneChildren()` - is either one better?

Reply:
🤷🏽 it was kind of an arbitrary decision. Using context is helpful in that, if you try to use the `ChatBubble` without the `ChatMessage`, it will throw an error. I have heard that cloning children isn't very efficient... but don't have solid info on that.

Reply:
Ok got it got it cool, yeah probably nice to have that error anyways